irods/manager/data_object_manager.py

```diff
@@ -534,6 +534,32 @@ def make_FileOpenRequest(**extra_opts):
             ret_value.seek(0,io.SEEK_END)
         return ret_value
 
+    def replica_truncate(self, path, desired_size, **options):
+            required_server_version = (4,3,3)
+            if self.sess.server_version < required_server_version:
+                raise ex.NotImplementedInIRODSServer('replica_truncate', required_server_version)
```

Comment:
This isn't *strictly speaking* true because replica_truncate exists and is usable in 4.3.2. However, https://github.com/irods/irods/issues/7918 describes why it's not as useful as it could be (the output string is empty on success even though it's supposed to contain information).

So, we should either keep this as it is with an explanatory comment to the effect of the one I left above, or we should change this to 4.3.2 and handle its attendant issues (if possible). I don't recall whether there was an issue with using the 4.3.2 endpoint here, but if it's enough of a problem to make it unusable, then leaving this as 4.3.3 with an explanatory comment is fine.

Just my two cents

Reply:
This seems resolved.